### PR TITLE
Castor online dqm updated config 74x

### PR DESCRIPTION
**Pull request by iik1997**
Changes introduced (and already tested on an online machine) in order to re-enable Castor online DQM

### CODE REVIEW

**Comment by iik1997** [comment]
@popovvp

**Comment by deguio** [comment]
@iik1997 could you please rebase this one as well?
thanks,
F.

**Comment by iik1997** [comment]
@deguio rebase done (hopefully correctly)

File: DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -73,72 +138,148 @@
...
+       showTiming = cms.untracked.bool(False),
+       dump2database = cms.untracked.bool(False),
+       pedestalsInFC = cms.untracked.bool(False),
+       # Define Labels
+       digiLabel = cms.InputTag("castorDigis"),
+       rawLabel = cms.InputTag("rawDataCollector"), #rawDataCollector #source
+       unpackerReportLabel = cms.InputTag("castorDigis"),
+       CastorRecHitLabel = cms.InputTag("castorreco"),
+       CastorTowerLabel = cms.InputTag("CastorTowerReco"),
+       CastorBasicJetsLabel = cms.InputTag("ak7CastorJets"),
```
Comment (deguio) [inline DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py]: in 73x I see that ak7BasicJets are used. is this wanted?

Comment (iik1997) [inline DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py]: I changed the label from ak7BasicJets to ak7CastorJets looking at the code here: https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/DQM/Integration/python/test/castor_dqm_sourceclient-live_cfg.py . In fact this is one of the "minor problems" I mentioned ("in passing") in one of my e-mails: unfortunately the CastorTowerJetMonitor (the one that uses the collections we are discussing) does not show anything sensible for example in https://cmsweb.cern.ch/dqm/online-playback/ . Should one try to fix it given https://github.com/cms-sw/cmssw/pull/7499 ?

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**